webpack.config.js

```diff
@@ -216,46 +258,25 @@ const adminConfig = {
         },
       ],
     }),
+    new webpack.ProvidePlugin({
+      process: 'process/browser',
+    }),
   ],
   optimization: {
-    runtimeChunk: {
-      name: 'vendor',
-    },
+    runtimeChunk: 'single',
     splitChunks: {
       cacheGroups: {
-        chunks: 'all',
-        default: false,
-        vendors: false,
-        vendor: {
-          name: 'vendor',
-          chunks: (chunk) => chunk.name === 'vendor',
-          priority: 1,
-          enforce: true,
-        },
-        admin_vendor_chunk: {
-          name: 'admin_vendor_chunk',
-          test: (module, chunks) => {
-            // add all modules from 'admin_vendor' entrypoint
-            if (chunks.some((chunk) => chunk.name === 'admin_vendor')) {
-              return true;
-            }
-
-            // add admin/form_editor_legacy/newsletter_editor shared modules
-            const filteredChunks = chunks.filter((chunk) => {
-              return ['admin', 'newsletter_editor', 'form_editor', 'settings'].includes(chunk.name);
-            });
-            return filteredChunks.length > 1;
-          },
-          enforce: true,
-          chunks: (chunk) => ['admin_vendor', 'admin', 'newsletter_editor'].includes(chunk.name),
-          priority: 0,
+        commons: {
```

Comment:
Looks like the custom `admin_vendor_chunk` was added to solve an issue with templates and translations being regenarated.
But I don't know how to check if this is still an issue. Maybe you have more context? https://github.com/mailpoet/mailpoet/commit/c934856f45d188b6074783b6cd002cd2e5918d74

Reply:
I don't have much more context. I haven't noticed any issues.

If I correctly understand [the original issue](https://github.com/webpack/webpack/issues/7230), it was happening when the bundle file was also an entry point. We use commons.js as a bundle with shared components and it isn't configured as an entry point and I think shouldn't be an entry point, but only contain modules that are used within some other entry points. I think that it is why we also need the admin_vendor entry point so that modules shared with the premium plugin are executed and exposed to the premium plugin.

Reply:
Thanks for taking the time to explain. Makes sense!